apps/crn-server/src/schema/autogenerated-schema.graphql

```diff
@@ -2731,6 +2743,7 @@ type EventsFlatDataDto {
   videoRecording: String
   """This box is automatically ticked if no output is added after 14 days from the event's end date."""
   videoRecordingPermanentlyUnavailable: Boolean
+  videoRecordingUpdatedAt: Instant
```

Comment:
whats all this doing here?

Reply:
Won't a rebase remove this?

Reply:
Just tried rebase and it didn't work. maybe reopen the PR?